examples/workflow/cloudflare-workers/wrangler.toml

```diff
@@ -0,0 +1,5 @@
+#:schema node_modules/wrangler/config-schema.json
+name = "qstash-workflow"
+main = "src/index.ts"
+compatibility_date = "2024-08-15"
+compatibility_flags = ["nodejs_compat"]
```

Comment:
why did we need the nodejs_compat flag?

Reply:
Wrangler added it when initting the project, I will test and remove if not needed